libautoscoper/src/gpu/cuda/RayCaster_kernels.cu

```diff
@@ -103,11 +89,11 @@ void volume_render(float* buffer, size_t width, size_t height,
 
     // Calculate the block and grid sizes.
     dim3 blockDim(32, 32);
-    dim3 gridDim((width+blockDim.x-1)/blockDim.x,
-                 (height+blockDim.y-1)/blockDim.y);
+    dim3 gridDim(((unsigned int)width+blockDim.x-1)/blockDim.x,
+                 ((unsigned int)height+blockDim.y-1)/blockDim.y);
```

Comment:
Should the outcome of the division be truncated to `unsigned int` ? The use of parenthesis seems off.

Also, could you confirm truncation is what we want here ?

See https://stackoverflow.com/questions/65225888/difference-among-static-cast-int-and-float2int-rn-in-cuda

Reply:
This is not performing truncation on the division, but converting the `size_t` width and height variables to prevent a compile time warning (see #42)
```
C:\SAM\Autoscoper\libautoscoper\src\gpu\cuda\Compositor_kernels.cu(70): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of data [C:\SAM\Autoscoper\build-test\Autoscoper-build\libautoscoper\libautoscoper.vcxproj] [C:\SAM\Autoscoper\build-test\Autoscoper.vcxproj]
```